test/pointing_party/vote_calculator_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule PointingParty.VoteCalculatorTest do
+  use ExUnit.Case, async: true
+  use ExUnitProperties
+
+  describe "calculate_votes/1" do
+    setup do
+      points_map = fixed_map(%{points: integer(1..5)})
```

Comment:
It would probably be better to use this, to match what the user does in the real app more closely:
```ex
points_map = fixed_map(%{points: member_of(0, 1, 3, 5)})
```

Reply:
The Elixir code uses `[0, 1, 3, 5]`. The HTML we create in the JavaScript uses `[1, 2, 3, 5]`. We should probably iron that out. Maybe `1..5` is fine for now, as long as we make a to-do to fix it before the workshop.